### PR TITLE
Fix | OLC-267 | PDF Image Upload Shows Blank in Template Builder

### PR DESCRIPTION
**Pull request by ZainAliOlm**
Added functionality to validate the file type before uploading

Jira: https://openletterconnect.atlassian.net/jira/software/c/projects/OLC/boards/5?assignee=712020%3Aac91928a-81ba-4d9e-b07c-36c0e80acade&selectedIssue=OLC-267

### CODE REVIEW

File: src/components/TemplateBuilder/index.tsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ import './styles.scss';
 // utils
 import {MESSAGES} from '../../utils/message';
 
-setUploadFunc(uploadFile)
+
```
Comment (aqdas95) [inline src/components/TemplateBuilder/index.tsx:46]: remove this empty line.

File: src/components/TemplateBuilder/index.tsx [hunk 2]
```diff
@@ -219,6 +220,15 @@ const TemplateBuilder: React.FC<TemplateBuilderProps> = ({ store, onReturnAndNav
     }
   }
 
+  const validateFile = async (file: File) => {
+    if (file && !allowedImageTypes.includes(file.type)) {
+      dispatch(failure('Only image files with extensions jpeg, png, or svg are allowed.'))
+      return;
+    } else {
```
Comment (aqdas95) [inline src/components/TemplateBuilder/index.tsx:227]: no need for else block since if and else both has returns.

Comment (aqdas95) [reply src/components/TemplateBuilder/index.tsx:227]:
```
     if (file && !allowedImageTypes.includes(file.type)) {
       dispatch(failure('Only image files with extensions jpeg, png, or svg are allowed.'))
       return;
     }
     return false;
```

File: src/components/TemplateBuilder/index.tsx (hunk 1 above)
Comment (ZainAliOlm) [reply src/components/TemplateBuilder/index.tsx:46]: No need. I have removed that function from this line.

**Review by aqdas95 — APPROVED** [review]

**Pull request merged**